postcss.config.js

```diff
@@ -1,13 +1,24 @@
+const cssnano = require("cssnano")({preset: "default"})
```

Comment:
tiny nit: I don't think these files are linted but can you add spaces around the `{}`?
```suggestion
const cssnano = require("cssnano")({ preset: "default" })
```

Reply:
i opted to use `cssnano-webpack-plugin` instead